### PR TITLE
[#176936458] Actuated Login Button so that it renders an Alert error …

### PR DESCRIPTION
**Pull request by DevorahLevi**
…message when one or both of the Login fields have been left blank.

### CODE REVIEW

**Review by Yatkins — APPROVED** [review]
Looks really good!

File: client/src/LoginPage.js [hunk 1]
```diff
@@ -1,22 +1,60 @@
...
+        alert = (
+            <Alert color="danger">
+                Error! Both the email and password fields must be filled in order to login. Please try again.
+            </Alert>
+        );
+
         return (
             <form>
 
+                {this.state.renderAlert && alert}
```
Comment (thecomputerguyjeff) [inline client/src/LoginPage.js]: 🥇 Great usage of short circuiting!!!

File: client/src/LoginPage.js [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
 export default class Login extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {email: "",
+                      password: "",
+                      renderAlert: false};
+    }
+
+    handleEmailFieldChange = (e) => {
```
Comment (thecomputerguyjeff) [inline client/src/LoginPage.js:13]: Love the es6 functions!

File: client/src/LoginPage.js [hunk 3]
```diff
@@ -1,22 +1,60 @@
...
+        if (email.localeCompare("") === 0 || password.localeCompare("") === 0) {
+            this.setState({renderAlert: true});
+        } else {
+            this.setState({renderAlert: false});
+            // Call backend API with user input
+        }
+    }
+
     render() {
+        alert = (
```
Comment (thecomputerguyjeff) [inline client/src/LoginPage.js]: I would define this outside the render function, or just use it directly in line 
this.state.renderAlert && <Alert> . ... </Alert>

**Review by thecomputerguyjeff — APPROVED** [review]
Excellent! One thing to think about... if they get a warning status...Then that warning will not be 'cleared' until the click the submit button again, rather than as soon as they start typing in the correct field. 

Don't change it in here, we'll make a separate story and prioritize appropriately. Approved.

File: client/src/LoginPage.js (hunk 3 above)
Comment (DevorahLevi) [reply client/src/LoginPage.js]: I will use it directly in-line. 
But if I had wanted to define it outside the render function, 'alert' would not have been recognized as a variable inside of the render function when I want to reference it. Would I have to add it to the state in the constructor? Or is there another way to have the alert variable be accessible to the render function?

**Comment by DevorahLevi** [comment]
@thecomputerguyjeff I pushed changes regarding your comments.

File: client/src/LoginPage.js (hunk 3 above)
Comment (thecomputerguyjeff) [reply client/src/LoginPage.js]: a function that returns the jsx component.

**Review by thecomputerguyjeff — APPROVED** [review]

**Pull request merged**